airbyte-integrations/connectors/destination-snowflake/src/test-integration/java/io/airbyte/integrations/destination/snowflake/SnowflakeTestUtils.java

```diff
@@ -41,9 +45,9 @@ public static List<JsonNode> dumpFinalTable(final JdbcDatabase database, final S
           AND table_name = ?
         ORDER BY ordinal_position;
         """,
-        unescapeIdentifier(databaseName),
-        unescapeIdentifier(schema),
-        unescapeIdentifier(table)).stream()
+        unescapeIdentifier(databaseName).toUpperCase(),
+        unescapeIdentifier(schema).toUpperCase(),
+        unescapeIdentifier(table).toUpperCase()).stream()
```

Comment:
This is a test util, so maybe it doens't matter, but I thought we wanted to let the DB evaluate the upper-case-ness of table names

Reply:
I overruled that decision https://airbytehq-team.slack.com/archives/C03C4AVJWG4/p1693495091006359 (but open to continued discussion!)

Reply:
Yes! 

> with snowflake uppercasing via not-quoting, we’re losing the ability to use special characters / reserved words in namespace/table/column names. That opens up a whole can of worms around escaping things, which I’d rather not deal with right now.

That makes a lot of sense.  👍

Reply:
this is still technically a user-facing decision (e.g. would they rather get `"FOO$BAR"`  or `FOO_BAR`). But that's a much smaller blast radius